### DISCUSSION TITLE
讲台上的计算机经常断网

### DISCUSSION THREAD

**Discussion by heroxbd** [physics-data/faq#24] [Q&A] [answered]
1. 试着看了网络配置，断网时 IP 地址还在
2. 断网时无法访问  auth4.tsinghua.edu.cn
3. 一般来说，五分钟之内会恢复

请问我应该排查其它什么地方？

**Comment by Harry-Chen** [comment, marked as answer]
1. 多喝热水
2. 重启试试
3. 拨打 62784859 反映问题

**Comment by jiegec** [comment]
1. Do not GuGuGu

**Comment by littlewu2508** [comment]
看看路由什么情况

**Comment by jiegec** [comment]
还要记得 auth6，不然就用 mirrors4
